core/src/link_resolver.rs

```diff
@@ -295,6 +295,10 @@ impl LinkResolverTrait for LinkResolver {
 mod tests {
     use super::*;
     use serde_json::json;
+    use std::env;
+
+    const MAX_IPFS_MAP_FILE_SIZE_VAR: &'static str = "GRAPH_MAX_IPFS_MAP_FILE_SIZE";
+    const MAX_IPFS_FILE_SIZE_VAR: &'static str = "GRAPH_MAX_IPFS_FILE_BYTES";
```

Comment:
How did this work before? 🤔

Reply:
They were at the top of the module, now they're only needed for tests! But the newest design allows for:

```rust
let mut env_vars = EnvVars::default();
env_vars.mappings.max_ipfs_file_size = 1337;
```

So these var. names are not there anymore.